tests/test_tmaUtils.py

```diff
@@ -345,6 +346,57 @@ def test_helperFunctions(self):
         self.assertEqual(slews, foundSlews)
         self.assertEqual(tracks, foundTracks)
 
+    def test_filterBadValues(self):
+        # test no bad values
+        # mean = median = 1.0
+        values = np.array([1.0, 0.96, 1.0, 1.04, 0.95, 1.0, 1.05, 1.0, 1.05, 1.0, 0.95])
+        mean = np.mean(values)
+        nReplaced = filterBadValues(values)
+        self.assertEqual(nReplaced, 0)
+        self.assertEqual(np.mean(values), mean)
+
+        # test with one bad values
+        values = np.array([1.0, 0.96, 1.0, 1.04, 2.95, 1.0, 1.05, 1.0, 1.05, 1.0, 0.95])
+        nReplaced = filterBadValues(values)
+        self.assertEqual(nReplaced, 1)
```

Comment:
Similarly, there ought to be at least one check that the expected elements were replaced with the expected values.